apps/website/pulumi/delivery.ts

```diff
@@ -76,6 +51,13 @@ class Delivery {
                 defaultTtl: 30,
                 maxTtl: 30
             },
+            customErrorResponses: [
+                {
+                    errorCode: 404,
+                    responseCode: 404,
+                    responsePagePath: "/_NOT_FOUND_PAGE_/index.html"
```

Comment:
This one does not fully work for my setup, but maybe I'm missing something:
![image](https://user-images.githubusercontent.com/2473784/145418942-20289adc-a86f-4b4b-ba0a-6d9513ae4b6b.png)